packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -194,13 +194,16 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
             execute: () => commands.executeCommand('editor.action.gotoLine')
         });
         commands.registerCommand({ id: 'actions.find' }, {
-            execute: () => commands.executeCommand(CommonCommands.FIND.id)
+            execute: () => commands.executeCommand('actions.find')
         });
         commands.registerCommand({ id: 'undo' }, {
             execute: () => commands.executeCommand(CommonCommands.UNDO.id)
         });
+        commands.registerCommand({ id: 'redo' }, {
+            execute: () => commands.executeCommand(CommonCommands.REDO.id)
```

Comment:
This seems incorrect. `redo` was missing, so the addition is OK, but I believe this should delegate into `redo` instead of `core.redo`.
@akosyakov, how can I verify this?